web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/ValidateSickLeavePeriodFacadeServiceImpl.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2021 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.service.facade.impl;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.common.lisjp.model.internal.Sjukskrivning.SjukskrivningsGrad;
+import se.inera.intyg.common.support.facade.model.value.CertificateDataValueDateRange;
+import se.inera.intyg.schemas.contract.Personnummer;
+import se.inera.intyg.webcert.web.service.facade.ValidateSickLeavePeriodFacadeService;
+import se.inera.intyg.webcert.web.service.fmb.FmbDiagnosInformationService;
+import se.inera.intyg.webcert.web.web.controller.api.dto.Icd10KoderRequest;
+import se.inera.intyg.webcert.web.web.controller.api.dto.MaximalSjukskrivningstidRequest;
+import se.inera.intyg.webcert.web.web.controller.api.dto.MaximalSjukskrivningstidResponse;
+import se.inera.intyg.webcert.web.web.controller.api.dto.Period;
+import se.inera.intyg.webcert.web.web.controller.facade.dto.ValidateSickLeavePeriodRequestDTO;
+
+@Service
+public class ValidateSickLeavePeriodFacadeServiceImpl implements ValidateSickLeavePeriodFacadeService {
+
+    private final FmbDiagnosInformationService fmbDiagnosInformationService;
+
+    @Autowired
+    public ValidateSickLeavePeriodFacadeServiceImpl(
+        FmbDiagnosInformationService fmbDiagnosInformationService) {
+        this.fmbDiagnosInformationService = fmbDiagnosInformationService;
+    }
+
+    @Override
+    public String validateSickLeavePeriod(ValidateSickLeavePeriodRequestDTO request) {
```

Comment:
Great that you are using var-keyword for local variables. See if you can be more consistent what you use. I personally prefer the final-keyword when I don't expect any other code to change the value.